src/Resources/contao/library/Contao/Database.php

```diff
@@ -706,6 +706,32 @@ public function getUuid()
 	}
 
 
+	/**
+	 * Quote the column name if it is a reserved word
+	 *
+	 * @param string $strName
+	 *
+	 * @return string
+	 */
+	public static function quoteColumnName($strName)
+	{
+		return \System::getContainer()->get('database_connection')->getDatabasePlatform()->quoteSingleIdentifier($strName);
```

Comment:
I wonder if we should just return ```'`' . $strName . '`'``` instead? Contao does not work with anything but MySQL and `quoteColumnName()` is potentially executed often, so we might be able to optimize performance here?

Reply:
I think we should definitely use `quoteSingleIdentifier()` from the doctrine database platform, but maybe we can store the DatabasePlatform object in a static variable so that we don’t call the whole `\System::getContainer()->get('database_connection')->getDatabasePlatform()` every time?

Reply:
We can simply use `quoteIdentifier()` instead of `quoteSingleIdentifier()`.

https://github.com/contao/core-bundle/commit/130fc001365bff41662c3dfea56b8a2961ae8969

Then we don't have to fix the `findInSet()` method, either.

Reply:
That makes sense 👍